src/wallet/init.cpp

```diff
@@ -11,6 +11,7 @@
 #include <util/moneystr.h>
 #include <util/system.h>
 #include <util/translation.h>
+#include <wallet/coincontrol.h> // for DEFAULT_AVOIDPARTIALSPENDS
```

Comment:
While here also add `#include <ui_interface.h>` because of `InitError`.

nit, does the comment really matter?

Reply:
Done.